build/tutorials/tutorials-contest/pangolin-token-subgraph/README.md

```diff
@@ -0,0 +1,480 @@
+        "amount": "46649696040618800000",
+        "from": "0xeed2db9b2d2645aaca044ecb397518ca6d9e74a5",
+        "id": "0x000296c30c325db75de48101737f5d54408af486ab021323225767701429e66e",
+        "to": "0xd7538cabbf8605bde1f4901b47b8d42c61de0367"
+      },
+      {
+        "amount": "1766396909474367721",
+        "from": "0xbb67987c040619842b0f8b0257bde63be842b27b",
+        "id": "0x000308522c9a2266f4ac73e13f02e7496decbad08286d52642ee768ff7ef343e",
+        "to": "0xd7538cabbf8605bde1f4901b47b8d42c61de0367"
+      },
+      {
+        "amount": "6826187345560538842",
+        "from": "0xa16381eae6285123c323a665d4d99a6bcfaac307",
+        "id": "0x0003aba174d1ad02e24d07122bb8bd66d194b640f79b19f3290885f2952b7b2b",
+        "to": "0x063b88d53d109c12ec21785c4e5e89bb71369432"
+      }
+    ]
+  }
+}
+```
+
+You can find the full code for the Pangolin Token subgraph in this [Github repository](https://github.com/ofemeteng/pangolin-token-subgraph).
+
+### Conclusion
+
+Congratulations on getting to the end of this tutorial. That was an extensive tour of The Graph and Avalanche ecosystems. In this tutorial, you were introduced to the concepts underpinning The Graph protocol and how it is set to bring about a new era of open APIs through which developers can build on the work of others in an open, decentralized, and permissionless manner. You were also introduced to GraphQL, the query language that powers The Graph protocol. The unique tweaks of The Graph protocol as relates to GraphQL were also highlighted and it was shown that The Graph currently only supports the Query type in the GraphQL specification. Next, you were introduced to the Avalanche blockchain, which was designed specifically to solve some of the scaling issues currently being experienced by other Layer 1 blockchain networks. Avalanche architecture and design choices were highlighted. Even though Avalanche operates based on a novel consensus algorithm, its Contract Chain which is one of 3 chains that make up the Avalanche ecosystem is EVM compatible. This means that developers from Ethereum and other EVM-based chains can use tools that they are already familiar with. Avalanche support for Solidity also means that DApps built for these chains can be deployed on Avalanche and immediately reap the benefits associated with low transfer fees and increased throughput. You were also shown how to use The Graph protocol to query data from Avalanche. The playground provided by The Graph was used to interact with the Pangolin decentralized exchange subgraph and you were also introduced to Apollo Client through which you can programmatically interact with subgraphs via your frontend projects. Finally, you built your own subgraph from scratch and deployed it on the hosted service by leveraging the governance token of Pangolin.
+
+At this point, you should be well equipped with the knowledge and tools you need to build awesome data-based projects using The Graph and Avalanche. Happy BUIDLing.
+
```

Comment:
Only have 1 line at the end of files.

Reply:
Files now have only 1 line at end of file.